AUTHORS

```diff
@@ -19,3 +19,4 @@ Guillermo López-Anglada <guillermo.lopez@outlook.com>
 Juan Pablo Paulsen <juanpablo.paulsen@gmail.com>
 Cameron Steffen <cam.steffen94@gmail.com>
 Rohan Vanheusden <rohan.vanheusden@gmail.com>
+Ivan Tugay <listepo@gmail.com>
```

Comment:
For better or worse, we've been keeping this list alphabetical by last name...  Could you scoot your name between Stefan and Rohan?